tests/service/predictors/srl_test.py

```diff
@@ -46,4 +46,11 @@ def test_cpu_vs_gpu(self):
             prediction_cpu = predictor_cpu.predict_json({"sentence": sentence})
             prediction_gpu = predictor_gpu.predict_json({"sentence": sentence})
 
-            assert prediction_cpu == pytest.approx(prediction_gpu)
+            assert set(prediction_cpu.keys()) == {"verbs"}
```

Comment:
This test is a bit redundant - we are checking that two different serialised models produce the same thing. Don't we want to be checking that loading a gpu trained model onto the cpu gives the same predictions as a gpu model loaded onto the gpu? Maybe i've got the wrong idea here but it seems a bit lucky that this is passing, and more to do with the fact that the model fixtures aren't trained very much.